controllers/api/eventRoutes.js

```diff
@@ -1,38 +1,176 @@
 const router = require('express').Router();
-const { Event } = require('../../models');
+const { Event, User, Task } = require('../../models');
+const sequelize = require('../../config/connection');
 const withAuth = require('../../utils/auth');
 
-router.post('/', withAuth, async (req, res) => {
```

Comment:
I really think we should go back to the async-await method so we don't have to chain a bunch of .then() statements together.  It will be harder to debug this way because our gold standard (the mini-project) is not this way.

Reply:
Ok. I will try and convert these back to async-await. If I encounter any issues, I will wait to push until tomorrow during our meeting.